apps/settings/js/panels/accessibility/panel.js

```diff
@@ -27,6 +29,12 @@ define(function(require) {
             _colorFilterDesc.setAttribute('data-l10n-id',
               enabled ? 'enabled' : 'disabled');
         });
+
+        SettingsListener.observe('accessibility.monoaudio.enable', false,
+          function(enabled) {
+            _audioDesc.setAttribute('data-l10n-id',
```

Comment:
So here I think just 'Disabled' and 'Enabled' is going to be misleading.
What happens is that the accessibility panel will have: Audio with the above description where what we want is 'Mono disabled' and 'Mono enabled' so please add them in the message bundle.